src/components/info-box/InfoBox.tsx

```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
   | 'info';
 
 export const InfoBox = ({
+  fillBg,
   variant,
   title,
   text,
   progressBar,
   icon,
 }: {
+  fillBg?: boolean;
   icon?: ReactNode;
   variant?: VARIANT_TYPES;
   title: string;
   text: string;
   progressBar?: {
     description?: string;
     level: number;
-    variant?: VARIANT_TYPES;
   };
 }) => {
   variant = variant || 'primary';
 
+  const iconClassName = 'info-box-icon ' + (fillBg ? '' : ('bg-' + variant));
```

Comment:
Can we increase readablity of these lines, It is hard to understand what is going on, maybe we can use back quotes (`)